doc/doxygen/install/install-linux.doxygen

```diff
@@ -50,7 +50,7 @@
   <ul>
     <li>
       Essential are \b gcc/g++ (>= 7.0) or similar ANSI C++ compiler with full C++17 support,
-      \b CMake (>= 3.1; with >= 3.9 suggested), \b Qt5, \b patch, \b autoconf (> 2.60), \b automake (> 1.9), \b libtool (libtoolize/glibtoolize).
+      \b CMake (>= 3.1; with >= 3.9 suggested), \b Qt5 (>= 5.6), \b patch, \b autoconf (> 2.60), \b automake (> 1.9), \b libtool (libtoolize/glibtoolize).
```

Comment:
Is the cmake version still correct? I thought it might be higher now

Reply:
Yes it's probably higher but I don't know how high. We can parse the Cmake minimum version statements but this does not help in checking since when Cmake has functional find modules